src/modules/tasks.js

```diff
@@ -0,0 +1,36 @@
+import { taskList, updateLocalStorage } from './data.js';
+import refreshList from './refresh.js';
+
+class Task {
+  constructor(string, bool) {
+    this.string = string;
+    this.bool = bool;
+    this.number = taskList.length;
+  }
+
+  addTask() {
+    taskList.push(this);
+  }
+
+  remove() {
```

Comment:
To follow the _Keep it Simple, Smart_ practice,  use `arrow functions` when possible. 💡